challenge_6/script.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+let dauphinsScores = [
+    [44, 23, 71],
+    [85, 54, 41],
+    ];
+
+let koalasScores = [
+    [65, 54, 49],
+    [23, 34, 27],
+    ];
+
+function calcAverage(array) {
+    const reducer = (previousValue, currentValue) => previousValue + currentValue;
+    let total = array.reduce(reducer);
+    let average = total / array.length;
+    return Math.round(average);
+}
+
+for (let i = 0; i < dauphinsScores.length; i++) {
+    console.log(calcAverage(dauphinsScores[i]));
+}
+
+for (let i = 0; i < koalasScores.length; i++) {
```

Comment:
T'aurais pu les mettre dans le même for. Tu aurais pu considérer que les équipes ont le même nombre de scores. Je cherche la petite bête